PMM-9249 documentation for dbstats (#649)
percona/pmm-doc: docs/setting-up/client/mongodb.md

- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use `MONGODB-X509` for SSL certificates.
- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use `$external` for SSL certificates.

#### Disable some collectors

By default, `mongodb_exporter` runs with all collectors enabled and discovering mode enabled. 
If you want to disable some collectors, use the `--disable-collectors` parameter.  
Example:  
```
pmm-admin add mongodb --disable-collectors=diagnosticdata,replicasetstatus
```

The valid collector names are: `diagnosticdata`, `replicasetstatus`, `dbstats` and `topmetrics`.  

#### Collect collStats and indexStats only for some collections.
Discovering mode will search for all databases and all colllections for collStats and indexStats. If you want to collect statistics only for some
collections use the `--stats-collections=db1.col1,db2.col2` to get stats only for the database.collections specified.

#### Prevent high resources consumption with discovering mode.

> If you have a high number of databases/collections, collecting stats for all of them could make the exporter use a high CPU percentage.

Author: ```suggestion
If you have a high number of databases/collections, collecting stats for all of them could result in high CPU consumption by the exporter.
```
